src/pages/lib-v1/push-notifications/testing/q/platform/[platform].mdx

```diff
@@ -0,0 +1,27 @@
+export const meta = {
+  title: `Testing`,
+  description: `Overview of testing your push notifications`,
+  filterKey: 'platform',
+  supportedPlatforms: INTEGRATION_FILTER_OPTIONS
+};
+
+import { generateStaticPaths } from '@/utils/generateStaticPaths.tsx';
+
+import { INTEGRATION_FILTER_OPTIONS } from '@/utils/filter-data.ts';
+
+export const getStaticPaths = () => {
+  return generateStaticPaths(meta.filterKey, meta.supportedPlatforms);
+};
+
+export const getStaticProps = (context) => {
+  return {
+    props: {
+      platform: context.params.platform,
+      filterKind: meta.filterKey
+    }
+  };
+};
+
+import testing from '/src/fragments/lib/push-notifications/common/testing.mdx';
```

Comment:
Intentionally targeting the shared `/lib/` fragment as changes to docs here should apply universally across versions

Reply:
By that you mean it'll be moved to a common folder and referenced both from lib and lib-v1?

Reply:
I mean I'll just leave it in `lib` even though it's referenced by `lib-v1` since it's applicable broadly and shared by the other platforms.